blocks/accordion/accordion.css

```diff
@@ -0,0 +1,69 @@
+.section.accordion-container {
+  padding-bottom: 36px;
+}
+
+.accordion details {
+  border-bottom: 1px solid var(--med-gray-color);
+  background: var(--text-color);
+  color: var(--background-color);
+  font-size: var(--heading-font-size-s);
+  font-weight: 900;
+}
+
+.accordion details p {
+  font-size: var(--heading-font-size-s);
+}
+
+.accordion details summary {
+  font-family: var(--bold-font-family);
+  position: relative;
+  padding: 0 16px;
+  padding-right: 48px;
```

Comment:
could combine these to one line - padding: 0 48px 0 16px

Reply:
unfortunately I seem to need "padding-right" in order to not break the CSS drawing of the chevron there.